pkgs/development/tools/libsigrok/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib
```

Comment:
Changes in this file are part of the sigrok-firmware-fx2lafw commit (e98efa6742cdfd531b14a2aa2323389a83770d5e) but belong in the libsigrok (dc21884dfaa971b6b5a3f5f74bd746e38166bb20) commit, which is almost empty at the moment.

Reply:
Argh! Guess I need to dig out the git skills to fix this.
Thanks for noticing!